js/index.js

```diff
@@ -1,10 +1,25 @@
 $(document).ready(function() {
 
-  $('a').click(function(){
+  $('.scroll-down').click(function(){
       $('html, body').animate({
           scrollTop: $( $.attr(this, 'href') ).offset().top
       }, 1200);
       return false;
   });
 
+  function showPopup() {
+    $(".popup").fadeIn();
+
+    $("body").addClass("overlay-background");
```

Comment:
I think we did this in class -- changing from a body class to using a "covering" div. If it didnt' make it into your actual project, it's worth doing.